test/unit/query/builder.js

```diff
@@ -40,6 +40,32 @@ const clientsWithNullAsDefault = {
   ),
 };
 
+var customLoggerConfig = {
+  log: {
+    warn: function(message) {
+      throw new Error(message);
+    },
+  },
+};
+var clientsWithCustomLoggerForTestWarnings = {
+  mysql: new MySQL_Client(
+    Object.assign({ client: 'mysql' }, customLoggerConfig)
+  ),
+  pg: new PG_Client(Object.assign({ client: 'pg' }, customLoggerConfig)),
+  'pg-redshift': new Redshift_Client(
+    Object.assign({ client: 'redshift' }, customLoggerConfig)
+  ),
+  oracledb: new Oracledb_Client(
+    Object.assign({ client: 'oracledb' }, customLoggerConfig)
+  ),
+  // sqlite3: new SQLite3_Client(
```

Comment:
Should this be commented out?

Reply:
there was no sqlite3 tests using sqlite3 a year ago.

maybe someone have missed test code so I did the same thing.

still some test does not include sqlite3, but it may be fixed in another PR.

anyway I un-commented it.